Some README edits (#548)
storj/storj: README.md

## Table of Contents

- [Contributing](#contributing-to-storj)
- [Installation](#installation)
- [Using via Storj CLI](#start-using-storj-via-the-uplink-cli)
- [Using via AWS S3 CLI](#start-using-storj-via-the-aws-s3-cli)
- [License](#license)
- [Support](#support)

# Contributing to Storj

> At the moment, all of our code here for v3 is open source. Have a code change you think would make Storj better? We are definitely open to pull requests. Send them along!

Reviewer: ```suggestion
All of our code for Storj v3 is open source. Have a code change you think would make Storj better? Please send a pull request along!
```

> Have comments, bug reports, or suggestions? Want to propose a PR before hand-crafting it? Jump on to [our Rocketchat](https://community.storj.io) to join the community and to talk to the Storj core team.

Reviewer: ```suggestion
Have comments, bug reports, or suggestions? Want to propose a PR before hand-crafting it? Jump on to [our Rocketchat](https://community.storj.io) and join the #dev channel to join the developer community and to talk to the Storj core team.
```

# Installation

### Install required packages

To get started running Storj locally, download and install the latest release of Go (at least Go 1.11) at [golang.org](https://golang.org).

You will also need [Git](https://git-scm.com/). (`brew install git`, `apt-get install git`, etc).
If you're building on Windows, you also need to install and have [gcc](https://gcc.gnu.org/install/binaries.html) setup correctly.

We support Linux, Mac, and Windows operating systems. Other operating systems supported by Go should also be able to run Storj.

### Download and compile Storj

> > **Aside about GOPATH**:  If you don't have a GOPATH set, you can ignore this > aside. Go 1.11 supports a new feature called Go modules,

Author: I didn't remove any content; I just moved this sentence to the front.

> > **Aside about GOPATH**:  If you don't have a GOPATH set, you can ignore this > aside. Go 1.11 supports a new feature called Go modules,

Author: Ah! Let's remove this sentence, then. Good catch.
